clickhouse-grpc-client/src/test/java/com/clickhouse/client/grpc/ClickHouseGrpcClientTest.java

```diff
@@ -205,4 +205,13 @@ public void testRowBinaryWithDefaults() throws ClickHouseException, IOException,
         throw new SkipException("Skip due to supported");
     }
 
+    @Test(dataProvider = "loadBalancingPolicies", groups = {"unit"})
+    public void testLoadBalancingPolicyFailover(ClickHouseLoadBalancingPolicy loadBalancingPolicy) {
+        throw new SkipException("Skip due to failover is not supported");
```

Comment:
The same as for CLI client.
I'm not sure about failover  here. Currently it does not work, so the tests are ignored. Please, let me know if failover should work for GRPC client as well